metadata.json

```
@@ -1,6 +1,6 @@
 {
     "shell-version": [
-        "45"
+        "45", "46"
```

Just a nit, can you use this syntax:

```suggestion
        "45",
        "46"
```